internal/jobworker/cgroup/cgroup_test.go

```diff
@@ -0,0 +1,233 @@
+package cgroup
+
+import (
+	"bufio"
+	"errors"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"testing"
+)
+
+func TestCleanup(t *testing.T) {
+	if !isRoot() {
+		t.Skip("must be root to run")
+	}
+
+	dir := t.TempDir()
+	service, err := NewService(WithMountPath(dir))
+	if err != nil {
+		t.Errorf("%+v", err)
```

Comment:
nit: Write clearer failure messages, specially for methods that are actually under tests (and are not just setup).

Reply:
I've updated messages to be more more informative, especially in this critical checks.